docs/Articles/XRTKUPMInstall.md

```diff
@@ -0,0 +1,96 @@
+
+1. [Download XRTK seed asset](https://github.com/XRTK/XRTK-Core/releases)
+2. [Import the XRTK Seed asset](#xrtk-seed-asset-install)
+3. Profit...
+
+For a little more detail, the sections below explain exactly what is happening.
+
+## Unity Packages
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_01_UnityPackages.png)
+
+Unity packages are separate to your project and act like references instead of raw code.  You can see them but not change them (at least not easily)
+This is where the XRTK (by default) installs safely and securely.
+
+## Unity Package Manager
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_02_PackageManagerMenu.png)
+
+You access the Unity Package manager in the editor from the *Unity -> Windows -> Package Manager* option in the menu, which brings up the package manager window.
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_03_PackageManager.png)
+
+This lists all the packaged (by default) that Unity provides out of the box, items ticked are those already installed.
+
+## XRTK Seed asset install
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_04_XRTKAutoInstallAsset.png)
+
+As you can see above, there is minimal code in the seed asset, as this isn't the XRTK itself, It is simply an enabler to register itself with the Unity package manager, once you've installed it, the UPM will automatically download the full XRTK and remove the seed package, ensuring there is nothing cluttering up your project.
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_05_XRTKDialog.png)
+
+# Manual Install
+
+If you prefer to register the XRTK with your project yourself, then you simply need to add the XRTK reference in to your project's ***Manifest.json***
+
+To locate the folder the manifest is in, simply right-click in your Unity project folder on the "Packages" folder as shown below, then select the "Show in Explorer" option:
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_06_LocatePackagesFolder.png)
+
+In the packages folder, you will see the specific Manifest.json file which you will need to open for editing using your favorite tool ([We prefer VSCode as it's awesome](https://code.visualstudio.com/))
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_07_ManifestJSON.png)
+
+Inside the Manifest file, you will see all the packages Unity provides "out of the box" listed:
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/Images/00_XRTKUPMInstall_08_ManifestDependandcies.png)
+
+To this you simply need to add the extra entry for the XRTK as follows:
+
+```json
+{
+  "dependencies": {
+      .....,
+    "com.xrtk.core": "https://github.com/XRTK/XRTK-Core.git"
```

Comment:
I suggest formatting this [like you did here](https://github.com/XRTK/XRTK-Core/pull/183/files#diff-e94668a25bf139d5b8256cf5126b08f8R35)